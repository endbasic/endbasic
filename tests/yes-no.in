true
TRUE
yes
YES
y
Y
n
false
FALSE
no
NO
n
N
y
