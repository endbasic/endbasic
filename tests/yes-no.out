Should we continue? Should we continue? Should we continue? Retry input: Invalid boolean literal 5
Should we continue? Should we continue? Should we continue? Should we continue? Should we continue? Should we exit? Should we exit? Should we exit? Should we exit? Should we exit? Retry input: Invalid boolean literal foo
Should we exit? Should we exit? Should we exit? 